src/app/Services/Restaurant.php

```diff
@@ -9,18 +9,26 @@
 use Infrastructure\Models\DeleteEntityJsonResponse;
 use Infrastructure\Models\GetEntityJsonResponse;
 use Infrastructure\Models\LoadCollectionJsonResponse;
+use Infrastructure\Models\SearchCriteria;
 use Infrastructure\Services\BaseService;
 use Symfony\Component\HttpFoundation\Request;
 
 class Restaurant extends BaseService
 {
     /**
+     * @Validation(name="id", type="string")
+     * @Validation(name="limit", type="string")
+     * @Validation(name="offset", type="string")
+     * @Validation(name="orderByASC", type="string")
+     * @Validation(name="orderByDESC", type="string")
+     * @param Request $request
      * @return LoadCollectionJsonResponse
      * @throws Exception
      */
-    public function load() : LoadCollectionJsonResponse
+    public function load(Request $request) : LoadCollectionJsonResponse
```

Comment:
It took me a while to understand that you filter request by validation and **cut** not mentioned fields in annotation. Basically only when I've started reading your validation class. Before I wrote comment that this is not secure.

So coming from here, it good to have such limitation for developer but this is not obvious and requires kind of documentation of framework to understand why request is missing some fields. Or debug or just knowledge that this works like this. 

Can you please add this to readme file to be aware of this nice move.

Reply:
I'll add a description

Reply:
Placeholder added to README